libs/expo/shared/ui-components/src/lib/ClientCard/ClientCard.tsx

```diff
@@ -1,24 +1,31 @@
 import {
+  IdCardOutlineIcon,
   LocationDotIcon,
   TentIcon,
   ThreeDotIcon,
+  UserOutlineIcon,
 } from '@monorepo/expo/shared/icons';
 import { Colors, Radiuses, Spacings } from '@monorepo/expo/shared/static';
+import { format } from 'date-fns';
 import { useRouter } from 'expo-router';
+import { HmisProfileType } from 'libs/expo/betterangels/src/lib/apollo/graphql/__generated__/types';
```

Comment:
@Davit-BetterAngels this import is incorrect and I'm not sure what it needs to be. Could you please take a look and update as needed?